probe-rs/src/architecture/arm/traits.rs

```diff
@@ -9,6 +9,8 @@ use super::{
     dp::{DpAddress, DpRegisterAddress},
 };
 
+pub(crate) mod polyfill;
```

Comment:
I am not sure polyfill is a good name. It's a bit too generic imo. If I read just this, I don't know what the file actually does.

Reply:
Well it polyfills the interface implementation for dumber probes, so technically ... 🤷‍♂️

Reply:
Okay, let's see :)